Jellyfin.Data/Entities/Artwork.cs

```diff
@@ -21,188 +9,194 @@
-            return (_Path = value);
-         }
-         set
-         {
-            string oldValue = _Path;
-            SetPath(oldValue, ref value);
-            if (oldValue != value)
+        }
+
+        /// <summary>
+        /// Backing field for Kind
+        /// </summary>
+        internal Enums.ArtKind _Kind;
+        /// <summary>
+        /// When provided in a partial class, allows value of Kind to be changed before setting.
+        /// </summary>
+        partial void SetKind(Enums.ArtKind oldValue, ref Enums.ArtKind newValue);
+        /// <summary>
+        /// When provided in a partial class, allows value of Kind to be changed before returning.
+        /// </summary>
+        partial void GetKind(ref Enums.ArtKind result);
+
+        /// <summary>
+        /// Indexed, Required
+        /// </summary>
+        [Required]
+        public Enums.ArtKind Kind
+        {
+            get
             {
-               _Path = value;
+                Enums.ArtKind value = _Kind;
+                GetKind(ref value);
+                return (_Kind = value);
             }
-         }
-      }
-
-      /// <summary>
-      /// Backing field for Kind
-      /// </summary>
-      internal global::Jellyfin.Data.Enums.ArtKind _Kind;
-      /// <summary>
-      /// When provided in a partial class, allows value of Kind to be changed before setting.
-      /// </summary>
-      partial void SetKind(global::Jellyfin.Data.Enums.ArtKind oldValue, ref global::Jellyfin.Data.Enums.ArtKind newValue);
-      /// <summary>
-      /// When provided in a partial class, allows value of Kind to be changed before returning.
-      /// </summary>
-      partial void GetKind(ref global::Jellyfin.Data.Enums.ArtKind result);
-
-      /// <summary>
-      /// Indexed, Required
-      /// </summary>
-      [Required]
-      public global::Jellyfin.Data.Enums.ArtKind Kind
-      {
-         get
-         {
-            global::Jellyfin.Data.Enums.ArtKind value = _Kind;
-            GetKind(ref value);
-            return (_Kind = value);
-         }
-         set
-         {
-            global::Jellyfin.Data.Enums.ArtKind oldValue = _Kind;
-            SetKind(oldValue, ref value);
-            if (oldValue != value)
+            set
             {
-               _Kind = value;
+                Enums.ArtKind oldValue = _Kind;
+                SetKind(oldValue, ref value);
+                if (oldValue != value)
+                {
+                    _Kind = value;
+                }
             }
-         }
-      }
+        }
+
+        /// <summary>
+        /// Required, ConcurrenyToken
+        /// </summary>
+        [ConcurrencyCheck]
+        [Required]
+        public uint RowVersion { get; set; }
 
-      /// <summary>
-      /// Required
-      /// </summary>
-      [ConcurrencyCheck]
-      [Required]
-      public byte[] Timestamp { get; set; }
+        public void OnSavingChanges()
+        {
+            RowVersion++;
+        }
```

Comment:
Where is the code that calls this?

Reply:
🤷 @EraYaN wrote that

Reply:
It's in the DbContext. It calls all these on all objects that are modified.